src/Core/Content/Seo/SalesChannel/SeoUrlAwareExtensionInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Seo\SalesChannel;
+
+use Shopware\Core\Framework\DataAbstractionLayer\Entity;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityCollection;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\EntitySearchResult;
+use Shopware\Core\Framework\Log\Package;
+
+#[Package('inventory')]
+interface SeoUrlAwareExtensionInterface
+{
+    /**
+     * @return array<string, EntitySearchResult<EntityCollection<Entity>>>
```

Comment:
I think we can simplify it as
 
```suggestion
     * @return array<string, Collection>
```